### DISCUSSION TITLE
Inav cannot arm from deck of the boat on sea.  Error Run-calibration on osd. Gyro not calibrated.

### DISCUSSION THREAD

**Discussion by mixasma** [iNavFlight/inav-configurator#1375] [General]
Inav cannot arm from deck of the boat on sea.  Error Run-calibration on osd. Gyro not calibrated.

**Comment by b14ckyy** [comment]
Yes. Works as designed. What's your question?

**Comment by MrD-RC** [comment]
Things like this should be put in **issues**, not **discussions**. 

However, as @b14ckyy pointed out, this is not a fault, iNav is doing what it is supposed to do. The gyro needs to calibrated from a flat (within reason) surface, that is not moving.

This is also not a configurator issue. It would be better in the main iNavFlight/inav repository.
